docs/cli/usage/plugin.md

```diff
@@ -183,7 +182,7 @@ You will be prompted to enter the plugin name, then select the plugin type, and
 The newly created plugin package is already added to the local Amplify CLI, so you can start testing it immediately.
 Let's say you have chosen to use the default plugin name: `my-amplify-plugin`
 
-```bash
+```console
```

Comment:
this should be bash and the next line should be console

Reply:
I don't think that's right. This is just for illustrative purposes and not for copying and pasting. Re-read the context and tell me if you disagree.